flink-staging/flink-gelly/src/test/java/org/apache/flink/graph/test/operations/GraphCreationITCase.java

```diff
@@ -54,16 +75,13 @@ public void testCreateWithoutVertexValues() throws Exception {
 		final ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
 		Graph<Long, NullValue, Long> graph = Graph.fromDataSet(TestGraphUtils.getLongLongEdgeData(env), env);
 
-        DataSet<Vertex<Long,NullValue>> data = graph.getVertices();
-        List<Vertex<Long,NullValue>> result= data.collect();
-        
+		graph.getVertices().writeAsCsv(resultPath);
```

Comment:
Oh... I made these changes before that pull request got merged. I change it now.